### ISSUE TITLE
Condition references aren't documented

### ISSUE THREAD

**Issue by benkehoe** [awsdocs/aws-cloudformation-user-guide#3]
In the page for Condition functions ( https://github.com/awsdocs/aws-cloudformation-user-guide/blob/master/doc_source/intrinsic-function-reference-conditions.md ), references to other conditions are shown as `{"Condition": "MyConditionName"}` and `!Condition MyConditionName`, but this function is not explicitly documented anywhere, nor is it mentioned on the main Conditions page ( https://github.com/awsdocs/aws-cloudformation-user-guide/blob/master/doc_source/conditions-section-structure.md )

Additionally, the section on resource attributes, which lists keys like CreationPolicy, doesn't mention the Condition key ( https://github.com/awsdocs/aws-cloudformation-user-guide/blob/master/doc_source/aws-product-attribute-reference.md ), *and* none those attributes or Condition are listed in the resource fields section of the template anatomy page on resources ( https://github.com/awsdocs/aws-cloudformation-user-guide/blob/master/doc_source/resources-section-structure.md )

**Comment by AWSRandall** [comment]
Hello, and thanks for using CloudFormation! 

I'm investigating your issue and will get back to you shortly. Thanks for your patience.

**Comment by talentedmrjones** [comment]
Last comment was >1year ago. Any word on this? As far as I can tell this is still undocumented.

**Comment by rachfop** [comment]
Thank you for telling us about this issue. It looks like "MyConditionName" has been removed from the docs to improve accuracy.

**Comment by benkehoe** [comment]
I don't believe you have addressed my issue, which was not related to the name. You have renamed `MyConditionName` to `SomeOtherCondition`, but as noted above, the reference syntax for `Condition` is not properly documented, nor is the `Condition` attribute of resources documented.

**Comment by rachfop** [comment]
@benkehoe thank you for your patience. I would hope you'd take a look at the [Condition](https://github.com/awsdocs/aws-cloudformation-user-guide/blob/master/doc_source/intrinsic-function-reference-condition.md)'s page on GitHub as we continue to use feedback like yours to improve our documentation.

**Comment by joshbean** [comment]
Closing this issue or pull request in advance of archiving this repo. For more information about the decision to archive this repo (and others in the "awsdocs" org), see [the announcement on the AWS News Blog](https://aws.amazon.com/blogs/aws/retiring-the-aws-documentation-on-github/).
